qlib/SqlUtil.qm

```diff
@@ -11437,6 +11444,11 @@ table.insertCommit(row);
         }
 
         #! @ref SqlUtil::AbstractTable::insertCommit() variant
+        /**
+            @param row a hash representing the row to insert; hash values can also be set with @ref sql_iop_funcs to insert values based on SQL operations to be used directly in the insert statement
+            @param sql an optional reference to a string to return the SQL generated for the insert statement
+            @param opt optional insert options; see @ref AbstractTable::InsertOptions for more info
```

Comment:
here and everywhere else in this file, can you please split the @param comments (the new ones you've added) to 120 per line? it should be possible to do it

Reply:
no, no new comment, just copy&paste of existing comment just to satisfy doxygen